_posts/2016/2016-07-04-ep6-offsite.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Team offsite meeting: workshops, focus, and fun"
+date: "2016-07-04 13:37:42"
```

Comment:
If you can make the changes today, we can leave the date, and I will release the post in the afternoon.